publish/build.gradle

```diff
@@ -5,12 +5,13 @@ publishing {
   publications {
     ltr(MavenPublication) {
       from components.java
+      afterEvaluate {
+        artifactId = 'elasticsearch-learning-to-rank'
+      }
```

Comment:
Jar is coming with name of project even if we specify artifactId in the script. ( for more detail, please see https://github.com/gradle/gradle/issues/11299)
To avoid that, force rename artifactId after generating script with afterEvaluate.